data/recommendation/style.scss

```diff
@@ -34,18 +34,16 @@ body {
     width: 100%;
   }
   .image-box {
-    width: 10%;
+    align-items: center;
+    display: flex;
```

Comment:
To do that, I'd put all 3 items in a flexbox, instead of creating 3 separate flexboxes.